pyhub/mcptools/microsoft/excel/com/decorators.py

```
@@ -0,0 +1,184 @@
+    if obj is not None:
+        try:
+            # Try to release the COM object
+            if hasattr(obj, '_oleobj_'):
+                obj._oleobj_.Release()
+        except:
+            pass
+
+        # Clear Python reference
+        try:
+            del obj
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)